src/etc/inc/filter.inc

```diff
@@ -2192,6 +2192,10 @@ function filter_nat_rules_generate() {
 		update_filter_reload_status(gettext("Creating outbound NAT rules"));
 		$tonathosts_array = filter_nat_rules_automatic_tonathosts();
 		$tonathosts = implode(" ", $tonathosts_array);
+
+		// Remove %radius from string
+		$tonathosts = str_replace('%radius,', '', $tonathosts);
```

Comment:
I don't believe this belongs here, but is required in the current implementation to remove invalid data.